java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -690,6 +690,286 @@ void testInnerJoinWithOnlyCommonKeys() {
     }
   }
 
+  @Test
+  void testLowerBound() {
+    boolean[] descFlags = new boolean[3];
+
+    try(ColumnVector cv = new ColumnVector(DType.INT64,
+          TimeUnit.NONE, 0, 0, null, null);
+        Table table = new Table(cv);
+        Table values = new Table.TestBuilder()
+            .column(20)
+            .build();
+        ColumnVector columnVector = table.lowerBound(false, values, descFlags)) {
+        fail("Should throw an assertion error for empty table");
+    } catch (AssertionError e) {
```

Comment:
you could use `assertThrows` instead, it should remove some of the boilerplate here.

Reply:
Will do!

Reply:
Done.